### ISSUE TITLE
recipe.run("tests/fixtures/papermage.pdf") process keep being killed

### ISSUE THREAD

**Issue by josephj1o4e1** [allenai/papermage#79]
My setting Windows10+WSL can't run: 

doc = recipe.run("tests/fixtures/papermage.pdf")
It seems to download the 13 pages 100% and then either killed off as a python script, 
or when in quick_start.ipynb mess up VScode that seems to be related to memory?

I'm not sure if this could be resolved.

**Comment by kyleclo** [comment]
Sorry, I'm not familiar with what would be required to run for Windows10+WSL.  Do you want to give this a try:
```
from papermage.recipes import MinimalTextOnlyRecipe

recipe = MinimalTextOnlyRecipe()
doc = recipe.run("tests/fixtures/1903.10676.pdf")
```

this is a single page PDF and the recipe is very very minimal.

**Comment by josephj1o4e1** [comment]
Yes, I tested on "test-uu.pdf" before for a single page case and it worked. 
It worked for the pdf you suggested as well. 

May I ask if there's a typical setting for using this package (RAM/OS)? 
I'm a windows user and it had the OSerror issue that appeared in previous issues . 
I switched to WSL and it worked fine only for less page pdfs. 

Thanks.

**Comment by kyleclo** [comment]
Ahh, unfortunately I can test this for MacOS Ventura (m1 macbook), but I don't have ability to test this for windows; this may be something you'll have to work out. 

As for memory, profiling it on my M1, I'm seeing CoreRecipe requires 2.2gb for a single page pdf, 2.4Gb memory for a 12 page pdf.  And MinimalTextOnlyRecipe requires 290mb for a single page pdf, 400mb for a 12 page pdf.

hope that helps?
